packages/gateway/test/setupJest.ts

```diff
@@ -34,10 +31,20 @@ const mock: IDatabaseConnector = {
     delete database[key]
     return keyExists ? 1 : 0
   }),
+  /** Doesn't implement TTL. Amend if needed. */
+  incrementWithTTL: jest.fn().mockImplementation(async (key) => {
+    database[key] ||= 1
+    ;(database[key] as number)++
+    return [database[key]]
+  }),
```

Comment:
Would love to use a actual Redis instance in tests or a proper mock, this way we only test our mock implementation